cmake/testing.cmake

```diff
@@ -289,8 +278,19 @@ if(F3D_PLUGIN_BUILD_ASSIMP)
   endif()
 endif()
 
-if(F3D_PLUGIN_BUILD_ALEMBIC)
-  f3d_test(NAME TestABC DATA suzanne.abc ARGS --load-plugins=alembic)
+if(F3D_PLUGIN_BUILD_EXODUS)
```

Comment:
What are those changes? It seems like you only moved lines

Reply:
alphabetical order, better to make sure we never forget a plugin. alembic, assimp, exodus, occt.